lib/llvmopencl/Workgroup.cc

```diff
@@ -1412,7 +1417,8 @@ Workgroup::createArgBufferWorkgroupLauncher(Function *Func,
         uint64_t ArgPos = ArgBufferOffsets[i];
         LLVMValueRef Offs = LLVMConstInt(Int32Type, ArgPos, 0);
         LLVMValueRef SizeByteOffset =
-            LLVMBuildGEP(Builder, ArgBuffer, &Offs, 1, "size_byte_offset");
+            LLVMBuildGEP2(Builder, LLVMTypeOf(ArgBuffer), ArgBuffer, &Offs, 1,
```

Comment:
This one might not get tested by our tests, but perhaps should be changed as well similarly to the above one